components/lib/inputnumber/InputNumber.js

```diff
@@ -58,7 +58,8 @@ export const InputNumber = React.memo(
                 currencyDisplay: props.currencyDisplay,
                 useGrouping: props.useGrouping,
                 minimumFractionDigits: props.minFractionDigits,
-                maximumFractionDigits: props.maxFractionDigits
+                maximumFractionDigits: props.maxFractionDigits,
+                roundingMode: props.roundingMode,
```

Comment:
Don't you need to add `roundingMode` to `InputNumberBase.js` as well?

Reply:
Thanks! I added default value for `roundingMode` in `InputNumberBase`.